WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -194,20 +281,11 @@ public final class WordPressComOAuthClient: NSObject {
                     guard let responseData = responseDictionary["data"] as? [String: AnyObject],
                           let userID = responseData["user_id"] as? Int,
                           let _ = responseData["two_step_nonce_webauthn"] else {
-                        failure(defaultError)
-                        return
+                        return failure(.unparsableResponse(response: response.response, body: response.data))
                     }
 
                     guard let needsMultifactor else {
-                        failure(
-                            NSError(
-                                domain: WordPressComOAuthClient.WordPressComOAuthErrorDomain,
-                                code: WordPressComOAuthError.unknown.rawValue,
-                                userInfo: [
-                                    NSLocalizedDescriptionKey: "Response requires handling the MFA Webauthn flow but handler given ('needsMultifactor' parameter)."
-                                ]
-                            )
-                        )
+                        failure(WordPressComOAuthError.missingMultiFactorHandler)
```

Comment:
Nice! This also fixes my mistake when writing that description string. It should have been "but **_no_** handler given"